libsplinter/src/circuit/template/mod.rs

```diff
@@ -38,6 +38,8 @@ pub(self) use crate::admin::messages::{CreateCircuitBuilder, SplinterServiceBuil
 
 /// Default file location for circuit templates
 pub const DEFAULT_TEMPLATE_DIR: &str = "/usr/share/splinter/circuit-templates";
+/// Environment variable for file location for circuit templates
+pub const TEMPLATE_DIR_ENV: &str = "TEMPLATE_DIR";
```

Comment:
We should prefix this with SPLINTER_, and make it a path instead of a single directory. 

SPLINTER_CIRCUIT_TEMPLATE_PATH=dir1:dir2:dir3

Reply:
Just to be clear here, you mean the `SPLINTER_CIRCUIT_TEMPLATE_PATH` would be something like `/usr/share/splinter` and then we would want to search there for the circuit template files? The second line formatting makes it look like you would be able to pass in multiple directories/paths like `SPLINTER_CIRCUIT_TEMPLATE_PATH=/usr/share/splinter:/tmp/lates:/foo/bar` and then all of those locations would be searched.

Reply:
Probably SPLINTER_CIRCUIT_TEMPLATE_PATH=/usr/share/splinter/circuit-templates:/tmp/lates:/foo/bar but otherwise yes, it should search the directories in the order given.